### PR TITLE
 #143412461 Redesigning the UI for user authentication

### PR DESCRIPTION
**Pull request by CeciliaWanjiku**
#### What does this PR do?
Re-design the UI for user authentication
 #### Description of Task to be completed?
As a user i should see a redesigned(according to material design specs and chosen color scheme) login/signup screen with the option to authenticate via facebook, twitter and google.
 #### How should this be manually tested?
User should sign-up or login
 #### What are the relevant pivotal tracker stories?
#143412461
 #### Screenshots (if appropriate)
screen shot 2017-05-15 at 6 57 03 pm
screen shot 2017-05-15 at 6 57 22 pm

### CODE REVIEW

**Review by muchai-mercy — APPROVED** [review]

**Comment by muchai-mercy** [comment]
Sign in with Github is working
Need to register as developer on: Google plus and twitter and Facebook as well
to make sure they are working

File: public/views/signup.html [hunk 1]
```diff
@@ -6,80 +6,80 @@
...
       </div>
     </div>
-    </div>
-    </div>
   </div>
-</div>
-
-
-
+</div>
```
Comment (LWanjiru) [inline public/views/signup.html]: Add new line here

**Review by LWanjiru — APPROVED** [review]
can be merged.

**Comment by gngchrs** [comment]
I need to be able to navigate to sign up from signin, and vice versa. current workflow is I have to go back to Home to do this. 

Fix/Add this and reopen.

**Pull request merged**